CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+
+A changelog is kept per crate, see:
+
+- [bitcoin CHANGELOG](bitcoin/CHANGELOG.md)
+
+- [addresses CHANGELOG](addresses/CHANGELOG.md)
```

Comment:
I did not look at the rendered version but is this empty line intentional?

Reply:
Yes I was also curios about that as well